tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -391,17 +397,8 @@ build_rootfs_distro()
 
 		engine_run_args+=" $(docker_extra_args $distro)"
 
-		# Relabel volumes so SELinux allows access (see docker-run(1))
-		if command -v selinuxenabled > /dev/null && selinuxenabled ; then
```

Comment:
Why is this removed?

Reply:
Sorry, I made a mistakes. This is still needed when we use `USE_DOCKER`, so I restored this.